src/firebase-rest.ts

```diff
@@ -75,10 +77,9 @@ export class Client {
 
     content = util.prettyJSON(content);
 
-    return request(options, content, this.debug)
-      .then(function(body: string) {
-        return body === '' ? null : JSON.parse(body);
-      });
+    return request(options, content, this.debug).then(function(body: string) {
```

Comment:
I think JS idioms is put the then on a new line. So it was right the first time. These days I would use a => instead of a function

Reply:
We have to support old versions that don't have support for `=>`

Reply:
typescript handles compiling to older version of JS

Reply:
Ha oops :) I'll fix it later.